### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by izakp**
Some minor fixes to the build / release pipeline, following up to https://github.com/artsy/hokusai/pull/177 cc @zephraph @dblandin

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,7 +12,8 @@ MINOR_VERSION ?= $(shell cat hokusai/VERSION | awk -F"." '{ print $$1"."$$2 }')
 
 dependencies:
 	pip install pipenv --quiet --ignore-installed
-	pipenv install --dev --deploy
+	pipenv install --deploy
+	pip install --quiet pyinstaller==3.3.1
```
Comment (zephraph) [inline Makefile]: Could `pyinstaller` just be added as a dev dependency? I'd actually thought I did that..

Comment (zephraph) [reply Makefile]: Yeah, it's here: https://github.com/artsy/hokusai/blob/c4828202a54bd11229883c4b9ed368bd9181327a/Pipfile#L7.

Comment (zephraph) [reply Makefile]: I see based on the commit message that this was intentional. I'm 👍 with it, but I would like to understand a bit more.

Comment (izakp) [reply Makefile]: Pyinstaller is not a dependency per-se - it's a build tool

Comment (izakp) [reply Makefile]: So it needs to be callable to create the "binary" distro, but is not part of Hokusai itself

**Pull request merged**